### PR TITLE
MCS-822 - Docs update

### PR DESCRIPTION
**Pull request by rartiss55**
Took a stab at creating a "quickstart" guide in the readme and moving the overall installation instructions/MCS config description to the prettier docs site. Note that some of the links may not work yet since I'm referencing some github files that haven't yet been moved to master to make sure the docs stay synced with the latest release.

### CODE REVIEW

File: docs/source/examples.rst [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
-========
+Example Usage
+=============
+
+Here are some example usage snippets for different use cases. Note that there are also additional scripts that may be useful in the scripts/ folder of the github project. 
+
+Using your own config file and/or unity build
+----------------------------------------------
+
+In this case, make sure your `MCS_CONFIG_FILE_PATH` path isn't set, as that takes precedence over the `config_file_path` parameter:
```
Comment (rartiss55) [inline docs/source/examples.rst:9]: I made a ticket to consider removing the environment variables later - I know people have complained about them anyway and with the pipeline the way it is now, we don't really need them.

File: docs/source/index.rst [hunk 2]
```diff
@@ -7,11 +7,12 @@ Machine Common Sense Documentation
 .. toctree::
     :maxdepth: 2
 
+    install
     examples
     api
-    training
     scenes
     schema
+    training
```
Comment (rartiss55) [inline docs/source/index.rst:15]: Maybe the order was fine before?

File: docs/source/install.rst [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+Mac Version:
+************
+
+`Download and unzip the Mac ZIP`_
+
+
+Addressables
+************
+
+As of 0.4.4, we are using addressable assets that are stored remotely on AWS, greatly decreasing the size of our Unity releases. This also means that there is a bit of a trade off on first-time start up when you download a new release, since the resources will have to be downloaded. If you would like to avoid this load time on initial start up (which could result in a timeout), you can download the latest release using the links above, and then run the `cache-addressables.py <https://github.com/NextCenturyCorporation/MCS/blob/master/scripts/cache_addressables.py>`_ script:
```
Comment (rartiss55) [inline docs/source/install.rst:96]: So this was written more from memory/asked Thomas about this yesterday --  I thought things worked fine after the first run if I didn't use the cache_addressables.py script, but I couldn't actually get a new addressables build to run properly through the Python API today. Didn't know if that was user error or the issues we are still having with the build.

File: docs/source/dev.rst [hunk 4]
```diff
@@ -133,15 +133,15 @@ Common examples of logging initialization:
     #Below initializes error only console logging
     mcs.init_logging(LoggingConfig.get_errors_only_console_config())
     
-Full List of Config Options
+More Config Options
 ---------------------------
 
-Outlined here is a comprehensive list of config file options that can be used.
+Outlined here is a list of config file options that can be used **in addition** to the ones listed under the :ref:`MCS Config File` section.
```
Comment (rartiss55) [inline docs/source/dev.rst:139]: It wouldn't let me have these duplicate sections in two places, so I split up the lists of config properties.

File: docs/source/index.rst (hunk 2 above)
Comment (deanwetherby) [reply docs/source/index.rst:15]: The only thing that feels out of place to me is the api. Maybe it goes after schema that way the docs flow from installation/setup -> example usage -> example scenes -> scene schema and then api?

File: docs/source/examples.rst (hunk 1 above)
Comment (deanwetherby) [reply docs/source/examples.rst:9]: Thanks for capturing that in a ticket. We are going to have to revisit configuration overrides shortly.

**Comment by deanwetherby** [comment]
I much prefer the succinct project README. It's just enough to get going.

**Review by deanwetherby — APPROVED** [review]

**Review by kdrumm — APPROVED** [review]
LGTM

File: docs/source/install.rst (hunk 3 above)
Comment (brianpippin) [reply docs/source/install.rst:96]: I think we need to tie a bow on this addressables stuff pronto it's all in a weird state, potentially the things i'm doing for Juan to test messed this up.

**Review by brianpippin — APPROVED** [review]
Looks good!

File: docs/source/install.rst (hunk 3 above)
Comment (rartiss55) [reply docs/source/install.rst:96]: Leaving this here for awareness: still need to investigate my local build, but this worked fine with the jenkins build/it ran so fast that I'm not even sure it was needed, so these instructions may be fine as is? 

I did add a ticket to incorporate the cache_addressables.py script a bit better at a later time or call it automatically (MCS-844).

Comment (rartiss55) [reply docs/source/install.rst:96]: (Tagging @erick-u and @JuanRMarquez-Ortiz for awareness)

**Pull request merged**